canonical-web-and-design/jaas-dashboard
# Switch region/cloud in status tables

## Done

Switch region/cloud in status tables

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Verify that region/cloud headers and columns match

## Details

Fixes #298
